addons/web/static/src/core/emoji_picker/emoji_picker.scss

```
@@ -39,12 +52,17 @@
     content: "\200b"; /* unicode zero width space character */
 }
 
-.o-EmojiPicker-search {
-    input {
-        &:not(:focus) {
-            & + .oi-search {
-                color: $text-muted;
-            }
-        }
+.o-EmojiPicker-search input {
+    &::placeholder {
+        color: var(--o-EmojiPicker__input-color, $input-placeholder-color);
+    }
+
+    &, &+i {
```

Missing empty spaces 
```suggestion
    &, & + i {
```